contracts/ETO/ETOTokenTerms.sol

```diff
@@ -85,14 +88,19 @@ contract ETOTokenTerms is Math, IContractId {
         EQUITY_TOKEN_NAME = equityTokenName;
         EQUITY_TOKEN_SYMBOL = equityTokenSymbol;
         EQUITY_TOKENS_PER_SHARE = equityTokensPerShare;
```

Comment:
> require(equityTokensPerShare > 0);

to avoid weird situations where 1 token is more than 1 share we could 

 ```require(equityTokensPerShare > EQUITY_TOKENS_POWER);```

Reply:
in the beginning I was a bit confused about with the precision......so here we do mean the smallest unit of the equity token per share here, right?  i'll refer to them as ```whole equity ``` and ```mini-equity``` from now on. OR WE ME

1 Share = EQUITY_TOKENS_PER_SHARE * mini-Equity

(now, a bit later, i do think it makes NEVER sense to talk about whole-equity-tokens!)

Reply:
I think you are right, but to avoid any doubts we have two scales here
1. scale of the equity token
2. scale of the share vs. equity token

shares are always integers
equity tokens have from 0 to 18 decimals (which is technically called SCALE and I incorrectly call it PRECISION which I will correct now - it's making too much mess)

1 Share == EQUITY_TOKENS_PER_SHARE 
EQUITY_TOKENS_PER_SHARE  = 10**n * 10**decimals

where n is set up by issuer to have nice price of a token ~1EUR. in case of Fifth Force we had `n` == 4, greyp has n==6

btw. we do not have check for n to be integer in EQUITY_TOKENS_PER_SHARE and I will add this now

Reply:
> I will add this now

I dont think you have added it yet. See this commment from me related to it: https://github.com/Neufund/platform-contracts/pull/330#discussion_r390266236